modules/op-starting-a-task-run-with-custom-config-map.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/unprivileged-building-of-container-images-using-buildah.adoc
+:_content-type: PROCEDURE
+
+[id="starting-a-task-run-with-custom-config-map_{context}"]
+= Starting a task run with custom config map
+
+After defining the custom Buildah cluster task, you can create a `TaskRun` object that builds an image as a `build` user with user id `1000`. In addition, you can integrate the `TaskRun` object as part of a `PipelineRun` object. 
+
+.Procedure
+
+. Create a `TaskRun` object with a custom `ConfigMap` and `Dockerfile` objects.
++
+.Example: A task run that runs Buildah as user id `1000`
+[source,yaml]
+----
+apiVersion: v1
+data:
+  Dockerfile: |
+    ARG BASE_IMG=registry.access.redhat.com/ubi8/ubi
+    FROM $BASE_IMG AS buildah-runner
+    RUN dnf -y update && \
+        dnf -y install git && \
+        dnf clean all
+    CMD git
+kind: ConfigMap
+metadata:
+  name: dockerfile <1>
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: buildah-as-user-1000
+spec:
+  serviceAccountName: pipelines-sa-userid-1000
+  params:
+  - name: IMAGE
+    value: image-registry.openshift-image-registry.svc:5000/test/buildahuser
+  taskRef:
+    kind: Task
+    name: buildah-as-user
+  workspaces:
+  - configMap:
+      name: dockerfile <2>
+    name: source
+----
+<1> In this example, we only want to run a TaskRun, so we won’t have any prior task that fetches some sources with a Dockerfile, so we will use a configmap instead.
+<2> Thanks to the workspace, we can mount a configmap as the source workspace for our buildah-as-user Task.
+
+. Start the task run or integrate it with a pipeline run.
```

Comment:
@sounix000 do we want / should we document using that task in a `PipelineRun` as well ? (That might be "closer" to real usage)

Reply:
@vdemeester : Yes, an example of PipelineRun would be great actually.

Reply:
For future reference:
- https://github.com/openshift-pipelines/docs/pull/3
- https://docs.openshift-pipelines.org/pipeline/unprivileged-builds.html#_pipelinerun